tests/integration-tests/tests/createami/test_createami/test_build_image_custom_components/image.config.yaml

```diff
@@ -6,7 +6,7 @@ Build:
     # Test script custom component with combination (ap-southeast-2, c5.xlarge, centos7) and (ap-southeast-2, c5.xlarge, ubuntu2004)
     {% if region == "eu-west-1" %}
     - Type: arn
-      Value: arn:aws:imagebuilder:eu-west-1:aws:component/stig-build-linux-high/2024.2.3/1
+      Value: arn:{{ partition }}:imagebuilder:eu-west-1:aws:component/stig-build-linux-high/2024.2.3/1
```

Comment:
nit: if eu-west-1 is hard wired, then no need to make partition dynamic. We could also make region dynamic